src/languages/en.ts

```diff
@@ -157,7 +157,7 @@ export default {
         lastName: 'Last name',
         phone: 'Phone',
         phoneNumber: 'Phone number',
-        phoneNumberPlaceholder: '(xxx)xxx-xxxx',
+        phoneNumberPlaceholder: '(xxx) xxx-xxxx',
```

Comment:
curious if this change is needed in this file

Reply:
this change was requested by @shawnborton [here](https://github.com/Expensify/App/pull/34498#issuecomment-1907636212)

Reply:
Thanks!